### PR TITLE
3 new templates

### PR DESCRIPTION
**Pull request by pedall**
- crying on the floor
- ugandan knuckles
- grumpy cat

### CODE REVIEW

**Comment by pedall** [comment]
@jacebrowning again something with Travis ;-)

File: data/templates/trumptable/config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: The Floor is Trump
+link: https://i.imgur.com/9GaaJ1Q.jpg
```
Comment (jacebrowning) [inline data/templates/trumptable/config.yml]: Can you provide a reference URL here? The point is to explain the source of the meme. I don't recognize this one.

File: data/templates/cryingfloor/config.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Crying on Floor
+link: https://i.imgflip.com/ysuv0.jpg
```
Comment (jacebrowning) [inline data/templates/cryingfloor/config.yml]: Can you provide a reference URL here? The point is to explain the source of the meme. I don't recognize this one.

File: data/templates/grumpycat/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name: Grumpy Cat
+link: https://imgflip.com/memegenerator/Grumpy-Cat
+default:
+- I hope that what does not kill you
+- Tries again
+aliases:
+- grumpy-cat
```
Comment (jacebrowning) [inline data/templates/grumpycat/config.yml]: `grumpy-cat` is redundant with `grumpycat`.

**Comment by pedall** [comment]
I updated the links of the other 2 memes (the guy that provided me the info said this are the link he had the pictures from). For the Trump picture he cropped the existing text. If those links are not enough I'll remove them completely.

**Comment by jacebrowning** [comment]
The reference link should be more than just an image. I want to hold off on adding those two until they become more established.

**Comment by pedall** [comment]
added another template :+1:

**Comment by pedall** [comment]
@jacebrowning removed trump since this was the only one left with a link to an image. The others know have knowyourmeme links

File: data/templates/grumpycat/config.yml (hunk 3 above)
Comment (jacebrowning) [reply data/templates/grumpycat/config.yml]: This line can be removed.

**Comment by jacebrowning** [comment]
Thanks! Please squash all this into one commit.

**Comment by pedall** [comment]
Is that what you meant? (im new to squashing ;-) )

**Comment by jacebrowning** [comment]
Not quite, but no worries. I can squash it from the GitHub UI.

**Pull request merged**